features/upgrade/etcd/upgrade.feature

```diff
@@ -0,0 +1,35 @@
+Feature: basic verification for upgrade testing
+  # @author geliu@redhat.com
+  # @case_id OCP-22606
+  @upgrade-prepare
+  @users=upuser1,upuser2
+  @admin
+  Scenario: etcd-operator and cluster works well after upgrade 
```

Comment:
needs suffix ` - prepare` so that Runner-v3 will work with this. We chatted about it.

Reply:
updated.